data/50-default.json

```diff
@@ -85,5 +85,26 @@
     "ref-kind": "runtime",
     "name": "com.endlessm.CompanionAppService.Locale",
     "branch": "stable"
+  },
+  {
+    "action": "install",
+    "serial": 2019102100,
+    "ref-kind": "app",
+    "collection-id": "com.endlessm.Apps",
+    "remote": "eos-apps",
+    "name": "com.hack_computer.Clubhouse",
+    "branch": "stable",
```

Comment:
Is this the correct branch? I don't think so – the default branch in `eos-apps` is `eos3`. Did you test that this file works? You can copy it into `/usr/share/eos-application-tools/flatpak-autoinstall.d/` and run `eos-updater-flatpak-installer --dry-run`.

Reply:
We have been using master/stable for Hack apps so far. We can move that to eos3 now that the Clubhouse is part of eos-apps. I am fine with that.

Thanks for the testing suggestion. I have tried with the master branch for now, but the filter does not work. With the current filter the installation is not scheduled. If I negotiate the amd64 rule it is scheduled.

Reply:
I think Flatpak calls the arch `x86_64`, sorry.

Reply:
Ahh, I compared with the archs in eos-image-builder. Ok that works now.

What about the branch, shall I keep stable? Or switch to eos3? I do not feel strongly at all.

Reply:
The app should be released as `eos3` like all other apps in `eos-apps`. We don't typically vary the branch between the dev / demo / prod repos.

Reply:
Ok, changed to use the eos3 branch for the Clubhouse.